etcdctl/command/mk_command.go

```diff
@@ -51,7 +51,12 @@ func mkCommandFunc(c *cli.Context, ki client.KeysAPI) {
 	ttl := c.Int("ttl")
 
 	ctx, cancel := contextWithTotalTimeout(c)
-	resp, err := ki.Set(ctx, key, value, &client.SetOptions{TTL: time.Duration(ttl) * time.Second, PrevExist: client.PrevIgnore})
+	// client.PrevIgnore would ignore the previous key.
+	// By passing client.PrevNoExist the Set method checks if
+	// the key had existed previously, and returns error when
+	// it tries to overwrite the key, which is how mk command
+	// is supposed to work.
```

Comment:
There is no need to mention `PrevIgnore` here because we don't use `PrevIgnore` here.

It would be better if you could make the second sentence more concrete.